### DISCUSSION TITLE
Backup bare-metal ceph-csi backed PVC to Azure BlobStorage

### DISCUSSION THREAD

**Discussion by pascalhobus** [vmware-tanzu/velero#6063] [Community support Q&A]
Hi community,

I've a bare-metal lab cluster consisting of:
- [Microk8s](https://microk8s.io/)
- [CephCSI](https://github.com/ceph/ceph-csi)
- [Kubernetes external snapshotter](https://github.com/kubernetes-csi/external-snapshotter)
- a dummy WordPress workload with a MariaDB

I've setup backups with Velero. I intented to create backups and store them to an Azure BlobStorage. I was following the instructions from the [azure plugin](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure) for Velero. Basically, the backup creation works, the backup gets created in the blob storage. Also the volumesnaphosts get created. What makes me bit sceptical, is that the volume snapshot is just a few bytes (instead of some 100MB which I would excpet), so I doubt that the volumes are correctly stored to the azure blob storage.

My question is now: is this use case anyways supported? Or is it meant that from a cluster in Azure I can create snapshots and store them to a blob storage within Azure - but not cross vendors (or bare-metal to Azure as I do)? I would like to understand supported scenarios by Velero, but I was not successful in finding this.

Thanks

**Comment by Lyndon-Li** [comment]
Velero v1.12 will be able to support this cross vendor backup-restore case, through the CSI volume snapshot data movement feature.

**Comment by draghuram** [comment]
@Lyndon-Li, Can't the user do the same even today with file system backups?

**Comment by Lyndon-Li** [comment]
@draghuram 
File system backup moves data from the volume(pvc/pv) itself but not from volume snapshot

**Reply by draghuram** [reply]
True. I thought that user just wants to migrate data which can be done even today with FSB, though data is read from live PVs. But perhaps, you are right in that user wants to take snapshot and move the data. This, as you suggested, would come in v1.12.
